students/biegon_piotr/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,82 @@
+print("Welcome to the awesome calculator :)")
```

Comment:
In previous review forgot to mention that there is no docstrings in whole file.

Every method should have docstring.
https://www.python.org/dev/peps/pep-0257/